src/unreadCards/DummyMessage.js

```diff
@@ -0,0 +1,65 @@
+/* @flow */
+// Renders dummy message.
+
+import React, { PureComponent } from 'react';
+
+import renderHtmlChildren from '../html/renderHtmlChildren';
+import htmlToDomTree from '../html/htmlToDomTree';
+import MessageFull from '../message/MessageFull';
+
+// Dummy data
+const message = {
+  avatar_url:
+    'https://secure.gravatar.com/avatar/37641655de3d5700215259f7bbefe3cd?d=identicon&version=1',
+  client: 'website',
+  content: '<p>Great talk to you then</p>',
+  content_type: 'text/html',
+  display_recipient: [],
+  flags: ['read'],
+  id: 264613,
+  reactions: [],
+  recipient_id: 1959,
+  sender_email: 'neeraj.wahi@gmail.com',
+  sender_full_name: 'Neeraj Wahi',
```

Comment:
Let's use fake names etc.. for the dummy data when it ends up in the codebase

Reply:
Would it be alright to use the email so that proper user avatar can be fetched? Or do we have a dummy user?